### ISSUE TITLE
DEV: Duplicated ENV Keys (HIGH PRIORITY)

### ISSUE THREAD

**Issue by niaalex** [hashmapinc/Drillflow#499]
EPIC: 
#361 Log CRUD Implementation 
Impact to: 
#498 
SIZE: 
Small
The client is still requesting this fix.  Please note below and address this as a High Priority.

See below for detail FYI:
Shuping [3:50 AM]
@Toni'a Alexander and @Sukhendu Mukherjee, I tried to find the corrected ENV keys from both https://drillflow.readthedocs.io/en/latest/integration/dot/envvars.html and https://hub.docker.com/r/hashmapinc/drillflow#documentation but failed to find keys for log related endpoints from both. So I refer to code https://github.com/hashmapinc/Drillflow/blob/master/df-server/src/main/resources/application.yml (master branch as I didn't find a 0.13.2 dedicated branch or tag), and I found the duplicated ENV keys are still there. Please refer to attached snapshot.  Please also confirm the `:mnemonic` in URL (highlighted in snapshot) is by design?

**Comment by TessForGithub2** [comment]
UPDATE: Sukhe investigated, and the issue is really a duplicated "script" header.
For example:        
log.channelset.path: ${LOG_PATH:https://api-demo.nam.drillops.slb.com/democore/channelreader/v4/witsml/channelSets}
        log.channel.path: ${LOG_PATH:https://api-demo.nam.drillops.slb.com/democore/channelreader/v4/witsml/channels}

The repetition is "LOG_PATH", and this is notation used within a configuration file, application.yml. I forget exactly how I used it at Chase, but Sukhe will investigate further.

I recommended a partial response to SLB, addressing those parts of the "ask" that we knew the answers for. He will run this past Toni'a before communicating with the client.
